.circleci/config.yml

```
@@ -0,0 +1,675 @@
+            - source-v1-
+      - checkout
+      - save_cache:
+          key: *source-cache
+          paths:
+            - .git
+      - run:
+         name: test-cftests-inference-part2.sh
+         command: export ORG_GRADLE_PROJECT_jdkTestVersion=25 && ./checker/bin-devel/test-cftests-inference-part2.sh
+
+
+# Unlimited fetchDepth (0) for misc_jobs, because of need to make contributors.tex .
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)